benchmarkapps/BenchmarkServer/BenchmarkServer.csproj

```diff
@@ -23,8 +21,8 @@
 
   <!-- These references are used when running on the Benchmarks Server -->
   <ItemGroup Condition="'$(BenchmarksTargetFramework)' != ''">
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="$(MicrosoftAspNetCoreAllPackageVersion)" />
```

Comment:
Hmm, don't change this stuff unless you know 100% that the benchmarks infrastructure works with it

Reply:
We killed Microsoft.AspNetCore.All. Benchmarks infrastructure will have to update.